views/peeps/index.erb

```diff
@@ -0,0 +1,40 @@
+<link rel="stylesheet" type="text/css" href="/stylesheets/chitter.css" />
```

Comment:
If you split this into two views, one for before logging in and one after, there would be a repetition of how you would  show the peeps across the two views, which violates the DRY principle and is a potential source of bugs since you'd have to consistently change both of these views if you wanted to change how peeps were displayed in the future.

Reply:
Ah, so this justifies why it is better practice to use the if statements within the views to display different things depending on session data. Thanks!